solarforecastarbiter/reference_forecasts/tests/test_main.py

```diff
@@ -327,7 +328,14 @@ def obs_5min_begin(site_metadata):
     return observation
 
 
-def test_run_persistence_scalar(site_metadata, obs_5min_begin, mocker):
+@pytest.fixture
+def session(requests_mock):
+    session = api.APISession('TOKEN')
+    requests_mock.register_uri('GET', session.base_url)
```

Comment:
to really test it, you should have the mock return the proper content for session to parse